From b44e2ee90bc91d385e0bd0458944fbf75ab717f5 Mon Sep 17 00:00:00 2001
From: "Hebbar, Gururaja" <gururaja.hebbar@ti.com>
Date: Thu, 3 May 2012 15:19:42 +0530
Subject: [PATCH 0944/1092] ARM: OMAP2+: am335x: Correct pin-mux string for
 MMC0

GPIO pin was used for Card detect signal for MMC0 Instance. MMC
structure was updated previously but the pin-mux structure was left out.

There was no error seen because the pin-mux value was higher(7) than
mode found(6). When both were OR'ed, correct value was taken and the
error was covered. This patch corrects the same.

This patch applies both for EVM, Beaglebone.

Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index bc3c363..970fb1b 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -598,7 +598,7 @@ static struct pinmux_config mmc0_wp_only_pin_mux[] = {
 };
 
 static struct pinmux_config mmc0_cd_only_pin_mux[] = {
-	{"spi0_cs1.mmc0_sdcd",  OMAP_MUX_MODE7 | AM33XX_PIN_INPUT_PULLUP},
+	{"spi0_cs1.gpio0_6",  OMAP_MUX_MODE7 | AM33XX_PIN_INPUT_PULLUP},
 	{NULL, 0},
 };
 
-- 
1.7.11.2

